.github/workflows/node.js.yml

```diff
@@ -10,6 +10,42 @@ on:
         branches: [master, feature/*, mynah-dev]
 
 jobs:
+    check-test-files:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+              with:
+                  fetch-depth: 2
+            - name: Check for the presence test files
+              id: check_file
+              run: |
+                  # Check if there are any changes in the src folder
+                  sourceFiles=$(git diff --name-only ${{ github.event.pull_request.base.sha }} | { grep 'src' || true; })
+                  if [ -z "$sourceFiles" ]; then
+                      echo "Did not find src files in the code changes"
+                      exit 0
+                  fi
+
+                  # Of those changes in the src folder, check if any of them are in *.test.ts files
+                  testFiles=$(echo $sourceFiles | { grep '\.test\.ts$' || true; })
+                  if [ -n "$testFiles" ]; then
+                      echo "Found test files in the code changes"
+                      exit 0
+                  fi
+                  echo "TEST_FILES_FOUND=true" >> $GITHUB_OUTPUT
+            - name: Notify contributor
+              if: steps.check_file.outputs.TEST_FILES_FOUND == 'true'
+              uses: actions/github-script@v7
+              with:
+                  github-token: ${{ secrets.GITHUB_TOKEN }}
+                  script: |
+                      github.rest.issues.createComment({
```

Comment:
Is there a way to update the existing comment instead of always creating a new one?

Reply:
In the newest changes the commits are now de-dupped